pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -541,7 +628,8 @@
     "runtime.win7-x64.runtime.native.System.IO.Compression": {
       "StableVersions": [
         "4.0.1"
-      ]
+      ],
+      "BaselineVersion": "4.3.0"
     },
     "runtime.win7-x86.runtime.native.System.Data.SqlClient.sni": {
```

Comment:
> We will need to decide if there are enough changes to warrant the cost of another TFS branch

I dont believe we have runway for this for preview build. But if this is warranted, we could look into for RTM. 

> we are going to need to work with you to move that code out of TFS.

@ianhays mentioned yesterday that there maybe an effort already inplace for this.